packages/server/graphql/types/PollOption.ts

```diff
@@ -0,0 +1,46 @@
+import {GraphQLID, GraphQLList, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import GraphQLISO8601Type from './GraphQLISO8601Type'
+import Team from './Team'
+import PollOptionId from '../../../client/shared/gqlIds/PollOptionId'
+
+const PollOption = new GraphQLObjectType<any, GQLContext>({
+  name: 'PollOption',
+  description: 'Poll options for a given poll',
+  fields: () => ({
+    id: {
+      type: GraphQLNonNull(GraphQLID),
+      description: 'Poll option id in a format of `pollOption:idGeneratedByDatabase`',
+      resolve: ({id}) => PollOptionId.create(id)
+    },
+    createdAt: {
+      type: GraphQLNonNull(GraphQLISO8601Type),
+      description: 'The timestamp the item was created'
+    },
+    updatedAt: {
+      type: GraphQLNonNull(GraphQLISO8601Type),
+      description: 'The timestamp the item was updated'
+    },
+    pollId: {
+      type: GraphQLNonNull(GraphQLID),
```

Comment:
this is where stuff gets dicey.
is this the numerical database ID or the GraphQL ID? e.g.  `1` or `PollOption:1`?
we'll have to be very explicit on which one it is in the description & make sure we always keep it consistent.
My advice is:
- any DB query helper should return the DB foreign key
- any dataloader should return the DB foreign key
- any graphql resolver should return the GraphQL ID

this will keep our database ID abstraction from leaking into the graphql schema. 
 that way, on our client cache, we'll still have globally unique IDs so we can do this: https://relay.dev/docs/guided-tour/reusing-cached-data/filling-in-missing-data/

Reply:
This is a very good catch! Added a better comment and a fix to provide a proper id. 🙏